### ISSUE TITLE
Quiz with random questions

### ISSUE THREAD

**Issue by dinism** [wjunkin/moodle-quiz_liveviewgrid#15]
**I made a quiz with 10 random question (Aleatória (PT) = random)**
![imagem](https://user-images.githubusercontent.com/14828021/58515768-cb2bba80-819d-11e9-86de-ff13bfe9a994.png)
**During the quiz i have this image. Nothing showing**
![imagem](https://user-images.githubusercontent.com/14828021/58515810-f3b3b480-819d-11e9-9844-3c97c6e9dfe7.png)
**The results of some students**
![imagem](https://user-images.githubusercontent.com/14828021/58515830-062dee00-819e-11e9-9f86-0efe598eba67.png)
**Is this correct?**
The last version in Moodle directory is 1.1.4
Thanks in advance

**Comment by wjunkin** [comment]
Hi, Dinism,
I have not pushed this version to the Moodle site. So the version here is 1.2.8 while the version on the Moodle site is still 1.1.4. I want to test this version out more completely. 
I have never tested out my program with the question being a random question. I will test this and see what I find out. (Evidently something is not working correctly with reandom questions.)

**Comment by dinism** [comment]
I'm using 1.2.8 and the tests are OK. Only issue is what i add here.

**Comment by wjunkin** [comment]
Thanks for your feedback. I'm gald everything is working except for the random question quiz. I'm afraid I won't be able to make Live Report work for the quiz with random questions. The structure of the Live Report is to have one column in the table for each question. If I modify this so that it works for a random question quiz, if the category from which the random questions are taken have 50 questions, then the table would have to have 52 columns (2 for the names and 50 for the questions) even if each student only answered 5 questions. 
In Live Report, if the quiz randmizes the order in which the student answers the questions, you will note that the order of the questions is the same order as the teacher sees the questions on the teacher interface. So, if the order of the questions is random, the teacher will see the students submitting answers into different columns in the table.
To have LIve Report handle quizzes that have random questions would require that I create code that is almost enrtirely new to be able to handle random questions. If I had the students' answers to the first question appear in the first column of the table (for example) the teacher could see the answer that each student gave to the first question but would not know what the question was.
So, in short, I don't plan to have Live Report be able to handle random questions. Sorry. Thanks for bringing this to my attention. I need to include this information in the documentation of the Live Report program. Because I don't plan to take care of random questions, I am now going to close this issue. 
Again, thanks.
